packages/modules/itwin-viewer-react/src/components/BlankViewer.tsx

```diff
@@ -0,0 +1,206 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import {
+  BlankConnectionProps,
+  ExternalServerExtensionLoader,
+  IModelApp,
+} from "@bentley/imodeljs-frontend";
+import { ErrorBoundary } from "@bentley/itwin-error-handling-react";
+import { UiFramework } from "@bentley/ui-framework";
+import React, { useEffect, useState } from "react";
+
+import Initializer from "../services/Initializer";
+import { getAuthClient } from "../services/ItwinViewer";
+import {
+  BlankConnectionViewState,
+  ExtensionInstance,
+  ExtensionUrl,
+  ItwinViewerCommonParams,
+  ItwinViewerUi,
+  ViewerExtension,
+} from "../types";
+import IModelLoader from "./iModel/IModelLoader";
+
+export interface BlankViewerProps extends ItwinViewerCommonParams {
+  blankConnection: BlankConnectionProps;
+  viewStateOptions?: BlankConnectionViewState;
+  extensions?: ViewerExtension[];
+}
+
+export const BlankViewer: React.FC<BlankViewerProps> = ({
```

Comment:
a lot of this code is duplicated from \packages\modules\itwin-viewer-react\src\components\Viewer.tsx
anyway we could simplify the initialization and extension loading to a single file? nbd, just something we'll need to be careful later when we update one of the viewers and not the other

Reply:
I added hooks for the theme and extension loading. I'm hesitant to move the initialization at this point because I feel like those could diverge and most things that are added would be needed by the Viewer and not the BlankViewer. If we get BlankViewer usage and find out differently, it's an easy task to do later

Reply:
sounds good, makes sense, thanks for the quick fix!